charts/ishare-satellite/templates/deployment.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "ishare-satellite.fullname" . }}
+  labels:
+    {{ include "ishare-satellite.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.deployment.replicaCount }}
+  revisionHistoryLimit: {{ .Values.deployment.revisionHistoryLimit }}
+  strategy:
+  {{- with .Values.deployment.updateStrategy }}
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "ishare-satellite.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
```

Comment:
I think it would be good to let the pod restart automatically on configmap changes, to make it easier to add new participants. See https://renehernandez.io/notes/rolling-pods-config-changes/ "Roll when Secret or ConfigMap resources changes"

Reply:
Annotation has been added